servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ProtocolCompatibilityTest.java

```diff
@@ -247,12 +247,26 @@ private static Collection<Arguments> sslAndCompressionParams() {
         return args;
     }
 
+    private static Collection<Arguments> statusMessageParams() {
+        final String[] messages = {
+                "abc", "Hello, World!", "a\r\nbc", "a%bc", "~ what? ~",
```

Comment:
Consider adding messages with characters that result in 2+ bytes in UTF-8 format, like Cyrillic or other alphabets. We need to test compatibility with grpc-java in those cases

Reply:
added more